programs/rodeos/streams/rabbitmq.hpp

```diff
@@ -14,7 +14,7 @@
 namespace b1 {
 
 class rabbitmq : public stream_handler {
-   std::shared_ptr<eosio::reliable_amqp_publisher> amqp_publisher_;
+   std::unique_ptr<eosio::reliable_amqp_publisher> amqp_publisher_;
```

Comment:
I don't think it makes sense to use `reliable_amqp_publisher` here anymore.  You're adding workarounds to bypass most of the functionality that it provides.

Reply:
Yeah. I'll just create a transactional_amqp_publisher roughly based on reliable_amqp_publisher